model/task_lifecycle.go

```diff
@@ -1913,7 +1914,7 @@ func UpdateBuildAndVersionStatusForTask(ctx context.Context, t *task.Task) error
 			if err != nil {
 				return errors.Wrap(err, "getting context for tracing")
 			}
-			_, span := tracer.Start(traceContext, "version-completion")
+			_, span := tracer.Start(traceContext, "version-completion", trace.WithNewRoot())
```

Comment:
Since we don't usually start new root spans, can we include the rationale in a comment?

Reply:
done